_pages/coc.md

```diff
@@ -3,3 +3,48 @@ layout: page
+Administrators will attempt to resolve the conflict in a manner that is accessible to the rest of the devanooga community, such as in a public channel. If open dialogue puts the privacy of one or more of the parties at risk, parts or all of the conflict resolution may be carried out in private.
+
+## Conduct of Administrators
+devanooga is organized by Chattanooga-area members. We want to foster a community that is driven and shaped by the members. With that in mind:
+
+- Administrators never act alone. Every action taken by an administrator is discussed with the larger group of admins before the action is taken.
+- Administrators will do everything within reason to act transparently. In some cases, it may be necessary (for the privacy of members) to do things "behind closed doors". We will avoid this whenever possible.
+- Administrators can be removed from power. If they violate the CoC, or otherwise intentionally prevent the growth of the community, they can be removed by a majority vote of the other administrators.
+
+## Changes to the Code of Conduct
+This is a living document. If you would like to openly discuss changes to this CoC, we welcome you to do any of the following:
+- Start a dialogue with administrators and the community in the `#general` Slack channel.
+- Use [our contact form](/contact) to send a message directly to administrators.
+- Send administrators an email at <admin@devanooga.com>.
+- Start an issue on [this website’s Github repository](https://github.com/devanooga/devanooga.github.io).
+- Fork this website’s Github repository, commit your changes, and issue a Pull Request.
+
+## Use of the Slack Organization
+The use of the devanooga Slack Organization is a privilege we provide and maintain for users that are in good standing with the community. At any time, the admins may choose to revoke the privileges of a user to access the Slack Organization.
+
+We ask that users keep these additional rules in mind when using the devanooga Slack Organization.
+
+- Users are allowed only one account.
+- Users may not impersonate other users, celebrities, etc.
+- Try to find a relevant channel for a topic of discussion. For example, don't talk about your latest trip to the gun range in `#ruby`.
+- Political, religious, and otherwise sensitive or controversial discussions are welcome, but we ask that users keep such topics out of the #general and #random channels.
+- Users are welcome to create new channels for any topic. Administrators _may_ remove a channel if it is determined that it violates our CoC.
+- Bots are discouraged. If you would like to create a bot, please limit it to only one channel. Keep in mind that most users are wanting to interact with other people, not robots. We may remove bots if they are abused in any way.
```

Comment:
I would say that bots that are there for _silly/joke_ purposes are discouraged. Helpful bots (eg: build status bots, 3rd party software integration bots, etc.) IMO are *encouraged* depending on the channel.